support/scripts/mklinux.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+
+# Linux arm64 boot header
+# https://www.kernel.org/doc/Documentation/arm64/booting.txt
+LINUX_ARM64_HDR = {
+    "CODE0"                   : [0x0000000091005a4d, 0x04],  # Offset   0
+    "CODE1"                   : [0x0000000000000000, 0x04],  # Offset   4
+    "LOAD_OFFS"               : [0x0000000000000000, 0x08],  # Offset   8
+    "IMAGE_SIZE"              : [0x0000000000000000, 0x08],  # Offset  16
+    "KERNEL_FLAGS"            : [0x0000000000000000, 0x08],  # Offset  24
+    "RES2"                    : [0x0000000000000000, 0x08],  # Offset  32
+    "RES3"                    : [0x0000000000000000, 0x08],  # Offset  40
+    "RES4"                    : [0x0000000000000000, 0x08],  # Offset  48
+    "MAGIC"                   : [0x00000000644d5241, 0x04],  # Offset  56
+    "RES5"                    : [0x0000000000000040, 0x04],  # Offset  60
+}
+LINUX_ARM64_HDR_SIZE = 64
+
+# surely there's a pythonic way for that
+def align_up(v, a):
+    return (((v) + (a)-1) & ~((a)-1))
+
+def autoint(x):
+    return int(x, 0)
+
+def main():
+    parser = argparse.ArgumentParser()
+    description='Prepends image with linux arm64 boot header'
+    parser.add_argument("ram_base", type=autoint, help='ram base')
+    parser.add_argument("img_base", type=autoint, help='image base')
+    parser.add_argument("entry", type=autoint, help='entry point')
```

Comment:
Perhaps just personal taste. Is there a reason why you chose to instead offload the fetching of the symbol to a Makefile rule instead of doing so in the script with less arguments. As you've seen this is also how I proceeded in my scripts, I invoke `nm` through python so that the script is self contained and requires as little arguments as possible :thinking:.

Reply:
I actually consider that but in the end favored against it to avoid code duplication between scripts. The alternative would had been to abstract common code into a library, which I didn't like very much either. On the other hand I don't particularly like the current way either, as much as I don't like invoking `nm` from python, but that's something we won't avoid until we move these to C. Either way I don't have a strong feeling towards any approach.

Reply:
Copied the `get_sym_val` function from `mkefi.py` and reduced the arguments to raw binary and elf file. Btw, that was another reason I didn't like this approach, as I found passing addresses more elegant than passing raw binary + ELF (and I didnt' want to call objcopy from the script).

Reply:
I am fine with either, especially since we intend to delete them anyway in favor of C-written ones. :angel: